README.md

```diff
@@ -57,6 +57,9 @@
       <td>
         <a href="/docs/README.tr.md">Türkçe</a>
       </td>
+      <td>
```

Comment:
Can you move this to be next to the "Add +" `td`? 🙂 

There is no real method to the ordering, besides just keeping them the order in which they have been translated

Reply:
Like this? (Swapped "Add +" and "Nederlands" around.)

Reply:
Works for me!